src/ansys/templates/python/solution/hooks/post_gen_project.py

```diff
@@ -0,0 +1,14 @@
+from ansys.templates.utils import keep_files
+
+
+DESIRED_STRUCTURE = []
```

Comment:
This structure needs to contain the different files you wish your template to include:

```suggestion
DESIRED_STRUCTURE = [
    ".flake8",
    ".github/workflows/ci.yml",
    ...
]
```

It is a bit tedious, but it guarantees that only the files you decide are kept in the final rendered project.

Reply:
I see. I though we can pass an empty list which means nothing should be used. I will update the code accordingly.